gateway-service-api/src/main/proto/org/hypertrace/gateway/service/v1/log_event.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+option java_package = "org.hypertrace.gateway.service.v1.log.events";
+
+package org.hypertrace.gateway.service.v1.log.event;
+
+import "org/hypertrace/gateway/service/v1/gateway_query.proto";
+
+message LogEventsRequest {
+  // todo change this to nano
+  sfixed64 start_time_nanos = 1;
```

Comment:
hmmm - to keep this in nanos or millis? My gut is millis so this becomes more of an implementation detail (since the unit on the attribute itself will be handled generically), but I'm on the fence.

Reply:
Ok, I can do that. Currently I am just converting timestamp to nanos in gql request builder.